src/identity/Identity.ts

```diff
@@ -136,6 +331,38 @@ export default class Identity extends PublicIdentity {
     )
   }
 
+  /**
+   * @description Encrypts Asymmetrical using random secret key (UInt8Array)
```

Comment:
* Not sure about the grammar 🙃
* Maybe give some context about what the UInt8Array is (or if you prefer, don't mention this at all, the example might be sufficient)

Reply:
Updated and removed the Unit8Array.